core/sqf/monitor/linux/commaccept.cxx

```diff
@@ -33,6 +33,7 @@ using namespace std;
 #include <signal.h>
 #include <unistd.h>
 
+extern CCommAccept CommAccept;
```

Comment:
Global objects cause race conditions at exit time. Their destructors are called in non-deterministic order. Does this object depend on any other objects? Does its destructor traverse to any other objects? If so, then this will probably break at some time in the future.